aws/helper_functions.py

```diff
@@ -116,19 +71,76 @@ def download_best_model( bucket, s3_model_output, hpo_results, local_directory )
     """ Download best model from S3"""
     try:
         target_bucket = boto3.resource('s3').Bucket( bucket )
-        path_prefix = s3_model_output.split('/')[3] + '/' + hpo_results['BestTrainingJob']['TrainingJobName'] + '/output/'
+        path_prefix = s3_model_output.split('/')[-1] + '/' + hpo_results['BestTrainingJob']['TrainingJobName'] + '/output/'
```

Comment:
For all of the path manipulation that's going on in this function, I'd recommend using [pathlib](https://docs.python.org/3/library/pathlib.html), but we probably want to at least avoid `+` for string concatenation.

Reply:
@miroenev Did you miss this one during the update?